File: src/hexdigts.c
/* hexdigts.c -- hexadecimal digits, shared across multiple source files.

   This is free and unencumbered software released into the public domain.
   For more information, please refer to <http://unlicense.org>. */

#include "defines.h"

#ifdef __WATCOMC__
#pragma aux default "$hexdigts$*"
#endif  /* __WATCOMC__ */

#include "hexdigts.h"

#if DEFINE_LOCAL_DATA == 1

const char HEXDIGITS[16] = "0123456789ABCDEF";

#endif  /* DEFINE_LOCAL_DATA == 1 */

File: src/hexdigts.pas
(* hexdigts.pas -- Pascal declarations for "hexdigts" library.

   This file is for linking compiled object files with Pascal linker.
   It will be deleted in future when we rewrite the project in C.

   This is free and unencumbered software released into the public domain.
   For more information, please refer to <http://unlicense.org>. *)

unit
    hexdigts;

interface

(*$I defines.pas*)

(*$ifdef DEFINE_LOCAL_DATA*)

const
    HEXDIGITS: array [0..15] of char = '0123456789ABCDEF';

(*$endif*)

implementation

end.
